sushi-config.yaml

```diff
@@ -407,22 +408,6 @@ groups:
     name: Other information about pregnancy (not used by birth and fetal death use cases)
       - ObservationBirthPluralityOfPregnancy
 
-  BFDRExtensionsAdministrative:
-    name: Extensions
-    resources:
-      - AuxiliaryStateIdentifier1
-      - AuxiliaryStateIdentifier2
-      - CertificateNumber
-      - ExtensionDateFiledByRegistrar
-      - ExtensionDatereceivedByRegistrar
-      - ExtensionEncounterMaternityReference
```

Comment:
ExtensionEncounterMaternityReference is now at the bottom of FHIR artifacts in the misc pile

Reply:
Moved this, and cleaned up the other junk at the bottom.